src/delay.rs

```diff
@@ -17,71 +18,16 @@ impl McycleDelay {
     }
 }
 
-impl DelayUs<u64> for McycleDelay {
+impl DelayUs for McycleDelay {
+    type Error = Infallible;
+
     #[inline]
-    fn delay_us(&mut self, us: u64) {
+    fn delay_us(&mut self, us: u32) -> Result<(), Self::Error> {
```

Comment:
In the latest `embedded-hal` release candidate, [embedded-hal-1.0.0-alpha.11](https://docs.rs/embedded-hal/1.0.0-alpha.11/embedded_hal/delay/trait.DelayUs.html) this trait no longer returns a `Result`, nor has an associated `Error` type.

Reply:
Shouldn't we aim at [1.0.0-rc.1](https://docs.rs/embedded-hal/1.0.0-rc.1/embedded_hal/index.html)?

Reply:
> Shouldn't we aim at [1.0.0-rc.1](https://docs.rs/embedded-hal/1.0.0-rc.1/embedded_hal/index.html)?

Definitely, same story there: https://docs.rs/embedded-hal/1.0.0-rc.1/embedded_hal/delay/trait.DelayUs.html

Wasn't sure about the separation, so just went with the `alpha` line of releases.